.github/workflows/ts-release-app-washboard.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  workflow_call: {}
+
+jobs:
+  release:
+    name: washboard-ui
+
+    runs-on: ubuntu-latest
+
+    if: startsWith(github.ref, 'refs/tags/typescript/apps/washboard-ui/v')
+
+    steps:
+      - name: Download Asset
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16
+        with:
+          name: washboard
+
+      - name: Create Release
+        uses: ncipollo/release-action@2c591bcc8ecdcd2db72b97d6147f871fcd833ba5
+        with:
+          draft: false
+          prerelease: false
+          makeLatest: false
```

Comment:
NIT: it would be nice to do some detection here and enable `draft` and `prerelease`

There are many ways to do it -- but I think in wasmCloud we sometimes do a check to see if there is something *after* the `vx.x.x`.

Please feel free to handle this later though in a follow up or never -- if we're fine with making node pre-releases just regular releases on GH that's fine too

Reply:
added an action for this on the latest commit